tests/scripts/multi-node/build-rook.sh

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env bash
+set -e
+
+
+#############
+# VARIABLES #
+#############
+
+rook_git_root=$(git rev-parse --show-toplevel)
+rook_kube_templates_dir="$rook_git_root/cluster/examples/kubernetes/"
+
+
+#############
+# FUNCTIONS #
+#############
+
+function fail_if {
+  if [[ "$(uname -n)" != "k8s-01" ]]; then
+    echo "You must run the script from k8s-01"
+    exit 1
+  fi
+  if [[ "$(id -nu)" != "vagrant" ]]; then
+    echo "Wrong user, log as 'vagrant'"
+    exit 1
+  fi
+  if ! git rev-parse --show-toplevel &> /dev/null; then
+    echo "It looks like you are NOT in a Git repository"
+    echo "This script should be executed from WITHIN Rook's git repository"
+    exit 1
+  fi
+}
+
+function purge_rook_pods {
+  cd "$rook_kube_templates_dir"
+  kubectl delete -n rook pool replicapool || true
+  kubectl delete storageclass rook-block || true
+  kubectl -n kube-system delete secret rook-admin || true
+  kubectl delete -f kube-registry.yaml || true
+  kubectl delete -n rook cluster rook || true
+  kubectl delete thirdpartyresources cluster.rook.io pool.rook.io objectstore.rook.io filesystem.rook.io volumeattachment.rook.io || true # ignore errors if on K8s 1.7+
+  kubectl delete crd clusters.rook.io pools.rook.io objectstores.rook.io filesystems.rook.io volumeattachments.rook.io || true # ignore errors if on K8s 1.5 and 1.6
+  kubectl delete -n rook-system daemonset rook-agent || true
+  kubectl delete -f rook-operator.yaml || true
+  kubectl delete clusterroles rook-agent || true
+  kubectl delete clusterrolebindings rook-agent || true
+  kubectl delete namespace rook || true
+  cd "$rook_git_root"
+}
+
+function purge_ceph_vms {
+  instances=$(vagrant global-status | awk '/k8s-/ { print $1 }')
+  for i in $instances; do
+    # assuming /var/lib/rook is not ideal but it should work most of the time
+    vagrant ssh "$i" -c "cat << 'EOF' > /tmp/purge-ceph.sh
+    sudo rm -rf /var/lib/rook
+    for disk in \$(sudo blkid | awk '/ROOK/ {print \$1}' | sed 's/[0-9]://' | uniq); do
+    sudo dd if=/dev/zero of=\$disk bs=1M count=20 oflag=direct
+    done
+EOF"
+    vagrant ssh "$i" -c "bash /tmp/purge-ceph.sh"
+  done
+}
+
+  # shellcheck disable=SC2120
+function add_user_to_docker_group {
+  sudo groupadd docker || true
+  sudo gpasswd -a vagrant docker || true
+  if [[ $(id -gn) != docker ]]; then
+    exec sg docker "$0 $*"
+  fi
+}
+
+function run_docker_registry {
+  if ! docker ps | grep -sq registry; then
+    docker run -d -p 5000:5000 --restart=always --name registry registry:2
+  fi
+}
+
+function docker_import {
+  img=$(docker images | grep -Eo '^build-[a-z0-9]{8}/rook-[a-z0-9]+\s')
+  # shellcheck disable=SC2086
+  docker tag $img 172.17.8.1:5000/rook/rook:latest
+  docker --debug push 172.17.8.1:5000/rook/rook:latest
+  # shellcheck disable=SC2086
+  docker rmi $img
+}
+
+function make_rook {
+  # go to the repository root dir
+  cd "$rook_git_root"
+  # build rook
+  make
+}
+
+# NOTE (leseb): I'm letting this unsued
```

Comment:
i wouldn't think you need to include your alias in a note. `git blame` can show that if needed.

Reply:
as you want, this is fixed anyway :)